indi-eqmod/eqmodbase.cpp

```diff
@@ -1065,7 +1065,22 @@ bool EQMod::ReadScopeStatus()
         {
             if (!(mount->IsRARunning()) && !(mount->IsDERunning()))
             {
-                SetParked(true);
+                currentRAEncoder = mount->GetRAEncoder();
+                currentDEEncoder = mount->GetDEEncoder();
+                parkRAEncoder    = GetAxis1Park();
+                parkDEEncoder    = GetAxis2Park();
+                if( static_cast<int32_t>(parkRAEncoder - currentRAEncoder) != 0 || static_cast<int32_t>(parkDEEncoder - currentDEEncoder) != 0 ) {
```

Comment:
Use std::abs(a-b) > 0 || ...etc which is better

Reply:
Whats about 
```
if( parkRAEncoder != currentRAEncoder || parkDEEncoder != currentDEEncoder )
```
?